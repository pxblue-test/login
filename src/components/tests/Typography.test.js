import React from 'react';
import Text from '../Typography';

import renderer from 'react-test-renderer';

describe('Typography', function () {
  it('Text Renders', () => {
    const tree = renderer.create(
      <Text>Here is a sample text</Text>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('Text Renders with custom styles', () => {
    const tree = renderer.create(
      <Text style={{color: 'green', fontSize: 24}}>Here is a sample custom text</Text>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});