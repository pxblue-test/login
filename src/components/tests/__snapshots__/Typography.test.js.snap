// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Typography Text Renders 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#424e54",
        "fontSize": 16,
      },
      undefined,
    ]
  }
>
  Here is a sample text
</Text>
`;

exports[`Typography Text Renders with custom styles 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#424e54",
        "fontSize": 16,
      },
      Object {
        "color": "green",
        "fontSize": 24,
      },
    ]
  }
>
  Here is a sample custom text
</Text>
`;
